src/main/java/com/google/sps/servlets/InitializeServlet.java

```diff
@@ -0,0 +1,24 @@
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/initialize")
```

Comment:
what is this servlet going to be handling?

Reply:
It is where the initial entry is made on the server side. Sent on init of creative